apps/events/models.py

```diff
@@ -415,11 +415,7 @@ def _trigger(self):
             return
 
         with current_team(experiment_session.team):
-            experiment_session.ad_hoc_bot_message(
-                self.params["prompt_text"],
-                fail_silently=False,
-                use_experiment=self._get_experiment_to_generate_response(),
-            )
+            experiment_session.try_send_message(self.params["prompt_text"], fail_silently=False)
```

Comment:
Hmm, I don't think this will quite work. Since `self.params["prompt_text"]` is a prompt, we should be calling `ad_hoc_bot_message` to treat that as an instruction, otherwise it would be awkward for the user to receive a message saying "tell the user to make take a break" for instance.

Also, when we call `_bot_prompt_for_user`, the generated message is being saved whereas `try_send_message` only tries to send the message.